zvariant/src/signature.rs

```diff
@@ -83,7 +83,7 @@ impl<'b> std::ops::Deref for Bytes<'b> {
 ///
 /// [identifies]: https://dbus.freedesktop.org/doc/dbus-specification.html#type-system
 /// [`slice`]: #method.slice
-#[derive(Hash, Clone)]
+#[derive(Eq, Hash, Clone)]
```

Comment:
I think the rebase got messed up again. You remove this in the first commit, add it again in the second and remove it back in the last commit.